src/Main.java

```diff
@@ -3218,7 +3253,7 @@ else if (region == 0xa || region == 0xc) {
 		*/ 
                 return (inventory.contains(Items.BLUE_KEY_CARD) && inventory.contains(Items.RED_KEY_CARD) && canLift(inventory))
 			|| (difficulty >= Difficulty.MERCILESS && inventory.contains(Items.WARP_REMOTE)
-				&& canSuperLift(inventory) && inventory.contains(Items.JUMP_BOOTS));
+				&& canSuperGP(inventory) && canSuperLift(inventory) && inventory.contains(Items.JUMP_BOOTS));
```

Comment:
Same question as for the blue chest here, what are the red overalls required for here? (Again, I might just be forgetting something obvious)

Reply:
Same reasoning as above, it's meant to make the Double Bear Bounce just a little easier to succeed since you can stun the bears with the Red Overalls quake.